### DISCUSSION TITLE
All user(s) data easily exportable

### DISCUSSION THREAD

**Discussion by RubyEyedReaper** [jellyfin/jellyfin#11759] [General]
We need a feature for easy export user accounts/data

including:
- password seeds
- watch history
- specific user settings

 This would be incredibly helpful, especially when updates cause Jellyfin to break completely, 
What's worse is when it wont re-deploy after replacing the backup of your directory  🤪 so you need a complete fresh install resulting in the loss of watch history accounts for both myself and my friends and family. This export user feature and restore would alleviate these issues, ensuring seamless restoration of all user data after reinstallations

location could fit under dashboard > users > advanced user settings drop down

**Comment by crobibero** [comment]
Existing feature request: https://features.jellyfin.org/posts/299/easy-import-export-jellyfin-settings
